addons/website_event_sale/controllers/main.py

```diff
@@ -23,20 +23,31 @@ def _create_attendees_from_registration_post(self, event, registration_data):
         if not any(info.get('event_ticket_id') for info in registration_data):
             return super()._create_attendees_from_registration_post(event, registration_data)
 
+        event_ticket_ids = [registration['event_ticket_id'] for registration in registration_data]
```

Comment:
Maybe we should complement with `if registration.get('event_ticket_id')` just to be safe.